meta-resin-common/recipes-connectivity/resin-proxy-config/resin-proxy-config/resin-proxy-config

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+#
+# copy over redsocks configuration file, set proper
+# permissions and start redsocks
+#
+
+set -e
+
+. /usr/sbin/resin-vars
+
+if [ ! -f $CONFIG_PATH ]; then
+	echo "resin-proxy-config: $CONFIG_PATH does not exist."
+	exit 1
+else
+	echo "resin-proxy-config: Found config.json in $CONFIG_PATH ."
+fi
+
+REDSOCKSCONF=${CONFIG_PATH%/*}/system-proxy/redsocks.conf
```

Comment:
Maybe it's worth using https://www.freedesktop.org/software/systemd/man/systemd.unit.html#ConditionArchitecture= ConfitionPathExists so that the respective service doesn't start at all if the configuration doesn't exists.